cmd/keymasterd/app.go

```diff
@@ -411,6 +430,7 @@ func browserSupportsU2F(r *http.Request) bool {
 	if strings.Contains(r.UserAgent(), "Firefox/") {
 		return true
 	}
+	logger.Debugf(3, "browser doest NOT support u2f")
```

Comment:
This seems like a significant enough event that it should be at debug level 0. Is there a reason to push this so far down?